### PR TITLE
Remove STATSSUMMARY from Portal change log

### PR DESCRIPTION
**Pull request by smalenfant**
## What does this PR (Pull Request) do?

This simply removes the nightly STATSSUMMARY entry lines from the Portal change log.
No changelog or test necessary.

## Which Traffic Control components are affected by this PR?

- CDN in a Box
- Traffic Ops

## What is the best way to verify this PR?

Wait until Traffic stats writes the Daily Summary. Should not see any log entries for  STATSSUMMARY

## The following criteria are ALL met by this PR

- [ ] This PR includes tests OR I have explained why tests are unnecessary
- [ ] This PR includes documentation OR I have explained why documentation is unnecessary
- [ ] This PR includes an update to CHANGELOG.md OR such an update is not necessary
- [X] This PR includes any and all required license headers
- [X] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://www.apache.org/security/) for details)

### CODE REVIEW

File: traffic_ops/traffic_ops_golang/trafficstats/stats_summary.go [hunk 1]
```diff
@@ -154,8 +154,8 @@ func CreateStatsSummary(w http.ResponseWriter, r *http.Request) {
 	}
 
 	successMsg := "Stats Summary was successfully created"
-	logMsg := fmt.Sprintf("STATSSUMMARY: %v, ID: %v, ACTION: create stats_summary", *ss.StatName, id)
-	api.CreateChangeLogRawTx(api.ApiChange, logMsg, inf.User, inf.Tx.Tx)
+	// logMsg := fmt.Sprintf("STATSSUMMARY: %v, ID: %v, ACTION: create stats_summary", *ss.StatName, id)
```
Comment (mitchell852) [inline traffic_ops/traffic_ops_golang/trafficstats/stats_summary.go:157]: please delete. no need to leave cruft in the code. if this code is needed, easy enough to revert it using git.

Comment (smalenfant) [reply traffic_ops/traffic_ops_golang/trafficstats/stats_summary.go]: Thanks. I was just testing and looks like I left this in.

**Comment by smalenfant** [comment]
This worked when merged in 5.1.x. Verified that the changelog doesn't get the STATSSUMMARY entries.

**Review by ocket8888 — APPROVED** [review]
Works

**Pull request merged**